### PR TITLE
@fjalling Added delete request to timetable

### PR DESCRIPTION
**Pull request by fjalling**

### CODE REVIEW

File: AirLineAPI/Controllers/TimeTablesController.cs [hunk 1]
```diff
@@ -75,5 +75,28 @@ public async Task<ActionResult<TimeTable[]>> GetTimeTableByEndDestination(string
                 return this.StatusCode(StatusCodes.Status500InternalServerError, $"Database Failure: {e.Message}");
             }
         }
+        [HttpDelete("{timeTableId}")]
+        public async Task<ActionResult> DeleteTimeTable(int timeTableId)
+        {
+            try
+            {
+                var OldTimeTable = await _repository.GetTimeTableByID(timeTableId);
```
Comment (AndreMorad) [inline AirLineAPI/Controllers/TimeTablesController.cs]: Starta en variabel med liten bokstav

**Review by AndreMorad — APPROVED** [review]
👍

**Review by istorp — APPROVED** [review]
nice 👍

**Review by Irvper77 — APPROVED** [review]
snyggt 👍

**Pull request merged**